File: commands/reload.py
from modules.load_commands import load_commands


async def reload(context):
    """reload the bot's commands"""
    commands = context["COMMANDS"]

    commands.clear()
    new_commands = load_commands()
    commands.update(new_commands)
    print(f"Reloaded {len(commands)} commands.", flush=True)

File: commands/testing/reload.ts
import type { Command, Context } from "../../types/types.ts";
import loadCommands from "../../modules/loadCommands.ts";
import { commands } from "../../globals.ts";

const command: Command = {
  data: {
    name: "reload",
  },
  async execute(ctx: Context) {
    await loadCommands();

    ctx.message?.reply(`Reloaded ${ctx.commands.size} commands!`);
  },
};

export default command;
